jina/executors/indexers/dbms/keyvalue.py

```diff
@@ -16,6 +17,7 @@ def _get_generator(
         for id_ in ids:
             vecs_metas_list_bytes = super()._query([id_])
             vec, meta = pickle.loads(vecs_metas_list_bytes[0])
+            assert Document(meta).tags['tag_field'] is not None
```

Comment:
why is this?

Reply:
was used during debugging an issue. removed